bootstrap/values.yaml

```diff
@@ -5,4 +5,29 @@ application:
   name: open-management-portal
   url: https://github.com/rht-labs/open-management-portal-deployment.git
   path: applications/
-  ref: master
\ No newline at end of file
+  ref: master
+
+argo-cd:
+  enabled: false
+  installCRDs: false 
+
+  global:
+# Do we want this?
```

Comment:
The short answer is no, but let's discuss

Reply:
Discussed and agreed that this is the approach for now. Hoping for no Argo bugs and good PR/MR process/automation to control content being submitted in "management repos" 

@pabrahamsson would like to follow up with you on the latter - i.e.: process/automation control of repo content.